example/register_device/.vscode/settings.json

```diff
@@ -0,0 +1,38 @@
+{
+    "files.associations": {
```

Comment:
That doesn't seem right. I guess it should be more like this:
```
"files.associations": {
        "*.c": "cpp",
        "*.cpp": "cpp",
        "*.h": "cpp",
        "*.hpp": "cpp"
    }
```
    
But I would love to have someone familiar with VSCode have a look at this, @TheWildTree